consumatio/external/tmdb.py

```diff
@@ -33,6 +35,11 @@ def get_movie_images(self, movie_id):
         data = self.get_data(query, self.db)
         return movie_images_to_dict(data)
 
+    def get_movie_credits(self, movie_id):
+        query = f'https://api.themoviedb.org/3/movie/{movie_id}/credits?api_key=c4bd02adee6e73c4d17e69b039267c90'
```

Comment:
Please pull the latest changes from the `main` branch and adjust the `api_key` value accordingly.